src/kernel/ContinuityAdvElemKernel.C

```diff
@@ -119,27 +122,34 @@ ContinuityAdvElemKernel<AlgTraits>::execute(
     const int ir = lrscv_[2*ip+1];
 
     DoubleType rhoIp = 0.0;
+    DoubleType projTimeScaleIP = 0.0;
     for (int j = 0; j < AlgTraits::nDim_; ++j) {
       w_uIp[j] = 0.0;
       w_rho_uIp[j] = 0.0;
       w_Gpdx_Ip[j] = 0.0;
       w_dpdxIp[j] = 0.0;
     }
 
+    for (int ic = 0; ic < AlgTraits::nodesPerElement_; ++ic) {
+      const DoubleType r = v_shape_function_(ip, ic);
+      projTimeScaleIP += r / v_udiag(ic);
```

Comment:
This is really a question to educate myself. Can this go into the loop below? May be GPU's don't like it.

Reply:
No, we need a fully computed `projTimeScaleIP` before entering the next loop, it is being used within that loop at line 152